node-runner-cli/release_ymls/radix-fullnode-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
 #      - "core_ledger:/home/radixdlt/RADIXDB"
       - "./node-keystore.ks:/home/radixdlt/node-keystore.ks"
   nginx:
-    image: radixdlt/radixdlt-nginx:1.3.0
+    image: radixdlt/radixdlt-nginx:1.3.1
```

Comment:
We are removing the automated release of nginx. So we need not bump this up unless there is a new release done manually

Reply:
Thanks, reverted :)